tvb_build/deploy_with_keycloak/hbp_openshift/tvb_deployer.yml

```diff
@@ -32,33 +47,62 @@ objects:
       resources:
         requests:
           storage: "2Ti"
-  - kind: Pod
+  - apiVersion: v1
+    data:
+      .tvb.configuration: "ADMINISTRATOR_NAME=admin\r\nADMINISTRATOR_PASSWORD=1a1dc91c907325c69271ddf0c944bc72\r\nADMINISTRATOR_EMAIL=jira.tvb@gmail.com\r\nKEYCLOAK_CONFIGURATION=/home/tvb_user/keycloak_config/keycloak_web_config.json\r\nKEYCLOAK_WEB_CONFIGURATION=/home/tvb_user/keycloak_config/keycloak_web_config.json\r\nENABLE_KEYCLOAK_LOGIN=True\r\nTVB_STORAGE=/home/tvb_user/TVB_STORAGE/\r\nUSR_DISK_SPACE=5242880\r\nMATLAB_EXECUTABLE=/usr/bin/octave\r\nSELECTED_DB=postgres\r\nURL_VALUE=postgresql+psycopg2://tvb-postgres/${POSTGRESQL_DATABASE}?user=${POSTGRESQL_USER}&password=${POSTGRESQL_PASSWORD}\r\nWEB_SERVER_PORT=8080\r\nURL_WEB=http://127.0.0.1:8080/\r\nCLUSTER_SCHEDULER=oar\r\nMAXIMUM_NR_OF_THREADS=4\r\nMAXIMUM_NR_OF_OPS_IN_RANGE=2000\r\nMAXIMUM_NR_OF_VERTICES_ON_SURFACE=300000\r\nDEPLOY_CLUSTER=False\r\nLAST_CHECKED_FILE_VERSION=5\r\nLAST_CHECKED_CODE_VERSION=9339\r\nADMINISTRATOR_DISPLAY_NAME=Administrator"
```

Comment:
Pls use email `tvb.admin@codemart.ro` for admin in such official deploys.
What happens if in this config the admin changes from web GUI some settings (e.g MAXIMUM_NR_OF_THREADS)? is the text from here re-read when the instance is restarted ?

Reply:
We will need to change this yml file when we want settings changed.